ios/emurgo.xcodeproj/project.pbxproj

```diff
@@ -712,7 +712,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "export DEVELOPMENT_BUILD_CONFIGURATIONS=\"+(Staging.Debug|Debug)\"\nexport NODE_BINARY=node\n$NODE_BINARY ../node_modules/@sentry/cli/bin/sentry-cli react-native xcode ../node_modules/react-native-schemes-manager/lib/react-native-xcode.sh";
+			shellScript = "export EXTRA_PACKAGER_ARGS=\"--sourcemap-output $DERIVED_FILE_DIR/main.jsbundle.map\"\nexport SENTRY_PROPERTIES=../sentry.properties\n\nexport DEVELOPMENT_BUILD_CONFIGURATIONS=\"+(Staging.Debug|Debug)\"\nexport NODE_BINARY=node\n\n../node_modules/react-native-schemes-manager/lib/react-native-xcode.sh index.ts\n";
```

Comment:
This is reverted to previous values from `develop` branch

Reply:
you can `git checkout develop -- ios/emurgo.xcodeproj/project.pbxproj` on base branch and sync back into this

Reply:
I've changed this in the base branch